### PR TITLE
Only keep triples with a predicate and language tag we will use later (#97)

### PR DESCRIPTION
**Pull request by cgueret**

### CODE REVIEW

File: twine/strip/processor.c [hunk 1]
```diff
@@ -64,14 +70,35 @@ spindle_strip(twine_graph *graph, void *data)
...
 				}
 			}
-			if(!match)
+
+			// If the object is a literal we need to ensure it is using either
+			// no language or one that we use for the indexing
+			literal_ok = 1;
+			object = librdf_statement_get_object(statement);
+			if(librdf_node_is_literal(object) &&
+			  (lang = librdf_node_get_literal_value_language(object)))
```
Comment (rjpwork) [inline twine/strip/processor.c]: I hate the "assign in an if conditional" method of doing things.

Comment (rjpwork) [reply twine/strip/processor.c]: If it's something that needs to be checked, check it separately!

Comment (rjpwork) [reply twine/strip/processor.c]: This is premature optimisation GONE MAD. We're not writing a game, folks!

**Comment by cgueret** [comment]
This code has been tested with:
- https://github.com/bbcarchdev/acropolis/blob/develop/test/data/geonames.nq to verify that languages are filtered correctly
- https://github.com/bbcarchdev/acropolis/blob/develop/test/data/likeit.nq , https://github.com/bbcarchdev/acropolis/blob/develop/test/data/Iceland.nq and https://github.com/bbcarchdev/acropolis/blob/develop/test/data/shakespeare-sample.nq for some general testing on the predicates

**Pull request merged**